packages/next-app/src/app/stableswapsimulator/(components)/TokensDistribution.tsx

```diff
@@ -27,23 +27,21 @@ const colors = [
 export function TokensDistribution() {
   const { initialData, customData } = useStableSwap();
   return (
-    <div className="flex flex-col w-1/3 gap-y-2 text-slate12">
+    <div className="flex flex-col w-1/4 gap-y-4 text-slate12">
```

Comment:
Maybe the parent component has to have a max width and the charts have to work with the space they have because on smaller screens (eg. with sidebar from arc open) we get x-scroll

<img width="1470" alt="image" src="https://github.com/bleu-studio/balancer-pool-metadata/assets/63298341/8d50286c-4b6d-4441-b141-5cf77cffc179">

Reply:
humm! Lets open a issue to check this out